.github/workflows/container-image.yml

```diff
@@ -112,9 +113,7 @@ jobs:
         with:
           name: policy-server-${{ matrix.targetarch }}-sbom
           path: |
-            policy-server-${{ matrix.targetarch }}-sbom.spdx.json
-            policy-server-${{ matrix.targetarch }}-sbom.spdx.cert
-            policy-server-${{ matrix.targetarch }}-sbom.spdx.sig
+            policy-server-${{ matrix.targetarch }}-sbom
```

Comment:
I wonder if this works, or one needs a wildcard pattern to catch the 3 files ending in `.spdx.{json,cert,sig}`.

Reply:
nevermind, this is a directory, and the files are inside of it.